File: src/main/kotlin-graphql/io/github/graphql/resolver/CommitContributionsByRepositoryResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for CommitContributionsByRepository
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
interface CommitContributionsByRepositoryResolver {

    @Throws(Exception::class)
    fun contributions(commitContributionsByRepositoryTO: CommitContributionsByRepositoryTO, after: String?, before: String?, first: Int?, last: Int?, orderBy: CommitContributionOrderTO?): CreatedCommitContributionConnectionTO

}

File: src/main/scala-graphql/io/github/graphql/scala/resolver/CommitContributionsByRepositoryResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._

/**
 * Resolver for CommitContributionsByRepository
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait CommitContributionsByRepositoryResolver {

    @javax.validation.constraints.NotNull
    @throws[Exception]
    def contributions(commitContributionsByRepositoryTO: CommitContributionsByRepositoryTO, after: String, before: String, first: Option[Int], last: Option[Int], orderBy: CommitContributionOrderTO): CreatedCommitContributionConnectionTO

}
